content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -295,33 +295,17 @@ See the [Kubeadm Troubleshooting guide](/docs/setup/production-environment/tools
 The kubelet is now restarting every few seconds, as it waits in a crashloop for
 kubeadm to tell it what to do.
 
-## Configure cgroup driver used by kubelet on control-plane node
+## Configuring a cgroup driver
```

Comment:
the conflict here was:

```
<<<<<<< HEAD
{{< warning >}}
Matching the container runtime and kubelet cgroup drivers is required or otherwise the kubelet process will fail.
{{< warning >}}

See [Configuring a cgroup driver](/tasks/administer-cluster/kubeadm/configure-cgroup-driver) for more details.

=======
If you are using a different CRI, you must pass your `cgroupDriver` value to `kubeadm init`, like so:
\```yaml
apiVersion: kubelet.config.k8s.io/v1beta1
kind: KubeletConfiguration
cgroupDriver: <value>
\```

For further details, please read [Using kubeadm init with a configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)
and the [`KubeletConfiguration` reference](/docs/reference/config-api/kubelet-config.v1beta1/)

Please mind, that you **only** have to do that if the cgroup driver of your CRI
is not `cgroupfs`, because that is the default value in the kubelet already.

{{< note >}}
Since `--cgroup-driver` flag has been deprecated by the kubelet, if you have that in `/var/lib/kubelet/kubeadm-flags.env`
or `/etc/default/kubelet`(`/etc/sysconfig/kubelet` for RPMs), please remove it and use the KubeletConfiguration instead
(stored in `/var/lib/kubelet/config.yaml` by default).
{{< /note >}}

The automatic detection of cgroup driver for other container runtimes
like CRI-O and containerd is work in progress.

>>>>>>> upstream/master
```

@reylejano @tengqm also here i picked master because it seems that it points now to a dedicated page on how to set up kubeadm init config.

One problem i've noticed in [the dev-1.21 deployment for this page](https://deploy-preview-26153--kubernetes-io-vnext-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) is that:
there seems to be a nested container warning, which looks kinda ugly:

![image](https://user-images.githubusercontent.com/6294120/113505464-52d37480-953f-11eb-8735-6dbe9b7e6c5c.png)

and when you click in the [warning section](https://deploy-preview-26153--kubernetes-io-vnext-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configuring-a-cgroup-driver) on `Configuring a cgroup driver` it goes to a 404. Not sure if this is supposed to point to some content that hasn't been generated yet as part of the release or if it's actually broken. 

cc @sftim maybe you want to weigh in

Reply:
I think it should be one `Warning` section with multiple statements. I don't think the `Troubleshooting` and `What's next` is supposed to be within the `Warning` section. I can change this after this the sync merges and do another sync.

Reply:
Aside: I think that callout should be a caution not a warning (but it's borderline). Something to fix elsewhere I think.